### DISCUSSION TITLE
Pretext > build target not working??

### DISCUSSION THREAD

**Discussion by drchscott** [community/community#65919] [New to GitHub] [answered]
Sorry, very new here.  I was working through a video "https://pretextbook.org/doc/guide/html/tutorial-videos.html" to learn how to write and deploy pretext via github.  From time to time while editing in the codespace, the video suggests clicking doing a "pretext > build target" and then viewing it as a web site.  I made a few basic content edits and it worked fine.  After a handful of simple edits, I thought I was off to the races only to discover that after another edit or two, the "build target" was no longer popping up the "view target" dialog.  I see that I can access the view target through other menu items or I can go to the web site link directly and do a server pull reload but ... nothing.  None of my changes are showing up anymore.  I've logged out and back into github and started the codespace again and I see that my changes are in my main.ptx but still, doing a build target still is not giving me an updated site like it did before.

Heeeeeeeeelllllllp :-(

**Comment by drchscott** [comment, marked as answer]
Hmmm... so, I now realize that the issue was that I was using a syntactically incorrect naming inside an xml id.  Unfortunately, the pretext compilation / processing didn't give me any good information about it so I did not know that a pretext syntax error was my problem.  I'm now implementing a undo / redo process after very minor updates so I can catch any such syntax problems.  But, more robust debugging so that I can do more coding at a time would help.  I expect that is possible but as a nube, I'm just missing it.

**Comment by queenofcorgis** [comment]
Congrats on self-solving especially as a new user 🚀 

Also pat yourself on the back for taking the time to make your undo / redo process so you can learn - that is awesome! 

Appreciate the feedback and I'm happy to pass it along - share any other Codespaces feedback you may have in our [Codespaces Category](https://github.com/orgs/community/discussions/categories/codespaces).
